### ISSUE TITLE
pygenefindergui: "Draw features" does not set slider location value

### ISSUE THREAD

**Issue by ak-watson** [dmnfarrell/pathogenie#1]
When highlighting a set of features in the feature table and right-clicking to "draw features", the slider location value in the interactive sequence feature viewer is not updated.

This means that the slider stays at position 1 in the currently viewed genome, so trying to zoom in and out from the current view resets the position to the start of the genome. Similarly, it means you can't pan to the left or right of the feature selection, because the location resets to the start of the genome.

When zooming, the slider position is read from start = int(self.slider.value()). I don't think this value is updated when using "draw features".

I'd suggest setting self.slider.value to the range for the currently viewed features. Something like
self.slider.setvalue(new_location) (I'd attempt to submit a patch myself, but I haven't really used github much so not sure how the process works!)

Thanks!

**Comment by ak-watson** [comment]
I added a line that seems to have solved it.
In "def plot_feature" I added the line

        s.slider.setValue(data.start + ((data.end-data.start)/2))

I'll send a pull request from my fork (if that is the right way to do things?)

My fork is here: [https://github.com/ak-watson/pygenefinder](url)

**Comment by dmnfarrell** [comment]
Thanks. I made the edit myself without the pull request since I replaced the redraw line with the slider setvalue because redraw is triggered every time the slider is changed anyway. I didn't think anyone was using this yet. It's still at an early stage.

**Comment by ak-watson** [comment]
No problem. I was just having a quick look after coming across your blog post on making the simple genome browser. I was going to take some inspiration from that and add a widget to pan to a specific location in the genome by typing the value in a text box. I saw you had used the browser within pygenefinder and decided to check it out.

I had been trying to do something similar to run within a Jupyter notebook (I since came across your implementation of that within pybioviz). Looks like a really useful set of tools you are building up!
